R/categories.R

```diff
@@ -239,3 +239,70 @@ setMethod("lapply", "Categories", function (X, FUN, ...) {
     X@.Data <- lapply(X@.Data, FUN, ...)
     return(X)
 })
+
+#' Change the id of a category for a categorical variable
+#' 
+#' @param variable the variable in a crunch dataset that will be changed
+#' @param from the (old) id identifying the category you want to change
+#' @param to the (new) id for the category
+#' @return \code{variable} with an the new category id
```

Comment:
roxygen supports markdown now, so you can call that `variable` in backticks rather than the \tex stuff. Also, how about for language saying "`variable` with category `from` and all associated data values mapped to id `to`"